packages/sonataflow-deployment-webapp/README.md

```diff
@@ -78,7 +79,7 @@ To create a consumer app for local tests following the Quarkus Getting Started A
 Add the webjar version to the properties section of your pom.xml
 
 ```xml
-<slwtDeploymentWebapp.version>0.0.2</slwtDeploymentWebapp.version>
+<sonataFlowDeploymentWebapp.version>0.0.2</sonataFlowDeploymentWebapp.version>
```

Comment:
```suggestion
<sonataFlowDeploymentWebapp.version>0.32.0</sonataFlowDeploymentWebapp.version>
```
We should not keep updating this version but let's use the first version that will possibly be out.
Also, I think we should instruct the user to go to webjars website and make sure to use the latest version.